lib/cog/commands/history.ex

```diff
@@ -35,25 +35,26 @@ defmodule Cog.Commands.History do
   end
 
   defp parse_args([]), do: {:ok, {nil, nil}}
-  defp parse_args([hist_start]) do
+  defp parse_args([hist_start]) when is_binary(hist_start) do
```

Comment:
Totally not a big deal, but it probably makes sense to call this parameter a `hist_range`